ui/src/pages/user_account_page/modals/ChangePassword.tsx

```diff
@@ -0,0 +1,172 @@
+import { User, UserAttributes } from '../../../models/User'
```

Comment:
I think it is duplicate now? As ChangePasswordModal is also still in user_account_page?